Service/Deserializer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIClientBundle\Service;
+
+use Auto1\ServiceAPIClientBundle\Exception\Response\MalformedResponseException;
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointInterface;
+
+interface Deserializer
```

Comment:
lets keep the consistency and add the Interface prefix

Reply:
changed